src/core/ngx_syslog.c

```diff
@@ -14,6 +14,11 @@
     + (NGX_MAXHOSTNAMELEN - 1) + 1 /* space */                                \
     + 32 /* tag */ + 2 /* colon, space */
 
+#define NGX_SYSLOG_RFC5424_MAX_STR                                             \
+    NGX_MAX_ERROR_STR + sizeof("<255>1 1970-01-01T00:00:00.000Z ") - 1         \
+    + (NGX_MAXHOSTNAMELEN - 1) + 1 /* space */                                 \
+    + sizeof("nginx ") - 1 /* appname, space */                                \
+    + 32 /* pid */ + 5 /* -, space */
```

Comment:
The backslashes should be in the **79**th column...

Other than that the rest of code looks more or less reasonable.

Reply:
@ac000 Fixed it.

Are you willing to introduce a code formatter offically on this project? Or it has already been that only I don't know?
We should fix nitpicks with something code formatter like a pretter rather than you mention each time. It can be save waisting time.
It's quite difficult for us to memorize completely the lengthy coding style guide and to adopt properly that.